packages/app/src/app/mobile-placeholder.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+
+import React from 'react'
+import { Container, Logo, Wrapper } from '../../components/mobile-placeholder.tsx/styled'
```

Comment:
Why have you placed this file outside of the `src` directory?

Let's remove `.tsx` from the dir name BTW.

Reply:
Ok I've removed .tsx from the dir name and placed in src directory